ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/spanning-tree.md

```diff
@@ -77,19 +77,21 @@ STP mode: **mstp**
 ### Global Spanning-Tree Settings
 
 Spanning Tree disabled for VLANs: **105,202,505-506**
+
```

Comment:
I don't think we need newlines here. But may be double space at the end of every line to separate them. That looks a bit more readable, especially considering the L3 header.

Reply:
I didn't make any changes to this. I just reformatted the template. The new-line is already present in the old files.

Reply:
As long as happy with the preview, just resolve it. I think, part of the challenge is how markdown preview plugin works. On github it looks good even with the newlines.

Reply:
Yeah, I am removing those new-blank-lines, which I introduced.

Reply:
Thank you @ankudinov . Now I understood why you have asked for the double-spaces.

Reply:
This is how it looks with the empty-blank line
![image](https://user-images.githubusercontent.com/88734745/149148622-84b5c0ed-1cd5-4bb4-9b0d-a5ff106a04bc.png)

Reply:
I like your idea with two blank spaces. That looks more clean.

Reply:
I made few changes to this section. Let me know if the unordered list view looks good. This looks good to me as all the global config is now under the title without using double spaces or \ or <br> at the end of line.